specs/vep-1243-vdk-meta-jobs/README.md

```diff
@@ -71,6 +80,17 @@ Non-goals are only features, which a contributor can reasonably assume were a go
 One example is features that were cut during scoping.
 -->
 
+### Goals
+
+* **Introduce a vdk-meta-jobs plugin**
+
+* **Provide a way to manage job dependencies**
+    - Execute jobs in a strict order. For example, one wants to ingest data from two different sources
+      into the same database table and read all the data only when both ingests are successful.
```

Comment:
nit: maybe `data job concurrency` would be too complex to propose at this point -> like synchronisation on parallel workloads capability; 
May mention it somewhere further, to facilitate searches or lookups by keywords. We're working on public content, and it is being indexed for reaching people.

Reply:
I am not sure I understand the proposal here. Do you mean adding something simpler in the text with the semantics of `data job concurrency` in order to enhance our "SEO score".